Robot.java_03_14_17

```diff
@@ -1,418 +1 @@
- * functions corresponding to each mode, as described in the IterativeRobot
- * documentation. If you change the name of this class or the package after
- * creating this project, you must also update the manifest file in the resource
- * directory.
- */
-public class Robot extends IterativeRobot {
-
-		
-	public static final int WINCHMOTOR = 2;
-	//public static OI oi;
-	public static RobotDrive driveTrain;
-
-	//public static RobotDrive driveTrain;
-	public static Winch winch;
-	public static GreenLightSystem greenlight = new GreenLightSystem();
-	Command autonomousCommand;
-	SendableChooser<Command> chooser = new SendableChooser<>();
-	//Testing UI, probably should put all joystick and buttons in a separate OI class
-	public static Joystick stick = new Joystick(0);
-	public static Joystick stick2 = new Joystick(1);
-	//for the second joystick
-	//need to find the right port
-	//changed button names-Anya
-	public static JoystickButton TriggerButton = new JoystickButton(stick,1); 
-	public static JoystickButton SideButton = new JoystickButton(stick,2);
-	//changed button to 3 to optimize for joystick (was 8)-Anya
-	public static JoystickButton HighButton = new JoystickButton(stick,5);
-	public static JoystickButton LowButton = new JoystickButton(stick, 3);
-	
-	//TEST - Anya
-	public static JoystickButton RightButton = new JoystickButton(stick,4);
-	
-	//use Trigger Button and Side Button , togggleWhenPressed(final Command command) 
-	//to toggle between arcadeDrive and tankDrive
-	//use TopButton to turn the green light on and off
-	
-	public static String driveMode;
-	public static String cameraMode;
-	
-	public static Timer myTimer = new Timer();
-	//hk
-	public static ADXRS450_Gyro gyro = new ADXRS450_Gyro();
-	//hk
-	
-	
-	//Encoders
-	public static Encoder leftEncoder;
-	public static Encoder rightEncoder;
-	/**
-	 * This function is run when the robot is first started up and should be
-	 * used for any initialization code.
-	 */
-	//Anya
-	public static SpeedController left;
-	public static SpeedController right;
-	//drivesides 
-	
-	public static Spark leftSpark;
-	public static Spark rightSpark;
-	
-	double initialLeftEncoderRead;
-	double initialRightEncoderRead;
-	
-	public static String state; 
-	
-	@Override
-	public void robotInit() {
-		//oi = new OI();
-		chooser.addDefault("Default Auto", new ExampleCommand());
-		// chooser.addObject("My Auto", new MyAutoCommand());
-		SmartDashboard.putData("Auto mode", chooser);
-
-		winch = new Winch(WINCHMOTOR, 9);
-		//how to call: public void tankDrive(double x, double y){
-		//winch.tankDrive(stick.getRawAxis(2), 0);
-		
-		leftSpark = new Spark(0);
-		rightSpark = new Spark(1);
-		left = leftSpark;
-		right = rightSpark;
-		
-		leftEncoder = new Encoder(1, 2, true, Encoder.EncodingType.k4X);
-		rightEncoder = new Encoder(4, 3, false, Encoder.EncodingType.k4X);
-		
-
-		
-		UsbCamera cam0 = CameraServer.getInstance().startAutomaticCapture(0);
```

Comment:
This article looks like it has the instructions you need to get multiple cameras up and running so that you can switch between them in teleop mode.
http://wpilib.screenstepslive.com/s/4485/m/24194/l/708159-using-multiple-cameras